modules/cassandra-datastax/src/main/java/com/intuit/wasabi/cassandra/datastax/DefaultCassandraDriver.java

```diff
@@ -92,6 +92,10 @@ private void initialize() throws IOException {
                 builder.addContactPoints(nodes.toArray(new String[nodes.size()]))
                         .withRetryPolicy(DefaultRetryPolicy.INSTANCE);
                 builder.withPort(getConfiguration().getPort());
+                builder.withCredentials(
```

Comment:
When cassandra does not require authentication, will empty username and password work?